Content.Server/Shuttles/Systems/ArrivalsSystem.cs

```diff
@@ -287,16 +298,20 @@ private void OnArrivalsDocked(EntityUid uid, ArrivalsShuttleComponent component,
     private void DumpChildren(EntityUid uid, ref FTLStartedEvent args)
     {
         var toDump = new List<Entity<TransformComponent>>();
-        DumpChildren(uid, ref args, toDump);
+        FindDumpChildren(uid, toDump);
         foreach (var (ent, xform) in toDump)
         {
             var rotation = xform.LocalRotation;
             _transform.SetCoordinates(ent, new EntityCoordinates(args.FromMapUid!.Value, Vector2.Transform(xform.LocalPosition, args.FTLFrom)));
             _transform.SetWorldRotation(ent, args.FromRotation + rotation);
+            if (_actor.TryGetSession(ent, out var session))
+            {
+                _chat.DispatchServerMessage(session!, Loc.GetString("latejoin-arrivals-dumped-from-shuttle"));
+            }
```

Comment:
```suggestion
                _chat.DispatchServerMessage(session!, Loc.GetString("latejoin-arrivals-dumped-from-shuttle"));
```
This localization doesn't currently exist in our Repo, you're going to need to add one in this PR.